docs/setting-up/client/index.md

```diff
@@ -16,6 +16,13 @@ You will need to have root access on the database host where you will be
 installing PMM Client (either logged in as a user with root privileges or be
 able to run commands with `sudo`).
 
+!!! alert alert-info "Note"
+
+    Credentials used in communication between the exporters and the PMM Server are the following ones:
+
+    * login is `pmm`
+    * password is equal to Agent ID, which can be seen e.g. on the Inventory Dashboard.
+
 **Supported platforms**
 
 PMM Client should run on any modern Linux 64-bit distribution, however
```

Comment:
We have a fixed list of supported platforms for PMM - https://www.percona.com/services/policies/percona-software-support-lifecycle#pt
Let's reffere to it

Reply:
It's at the bottom (seealso). I prefer to put all 'see also' blocks in at the bottom of the page rather than scatter them throughout as it breaks the flow for the reader. I alternatively link them inline or both.